src/Cpu/CpuArchitecture.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace DBrekelmans\BrowserDriverInstaller\Cpu;
+
+enum CpuArchitecture: string
+{
+    case X86_64 = 'x86_64';
+    case ARM64  = 'arm64';
+
+    public function toCommandOutput(): string
+    {
+        return match ($this) {
+            self::X86_64 => '',
```

Comment:
```suggestion
            self::X86_64 => 'x86_64',
```

Reply:
are you sure about this? It was intentional as it could be amd64...

Reply:
As far as I understand `x86_64` and `amd64` are one and the same, but I'm not super familiar with cpu architectures so please correct me if I'm wrong. Perhaps `amd64` is better, since it seems like the more commonly used name. I'd suggest renaming the constant as well as the string value here.

Reply:
ok